spec/factories/rehire_sites.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :rehire_site do
+    
```

Comment:
See https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#defining-factories for how to define a Factory for a rehire site.